grunt/config/json-minify.js

```diff
@@ -0,0 +1,7 @@
+module.exports = function (grunt, options) {
+    return {
+        build: {
```

Comment:
can you rename from ``build`` to ``minify`` so the ``minify`` task is responsible for both json and javascript?